### PR TITLE
Use 2018 edition.  Closes #2.

### LINKED ISSUE #2: 2018-ify code
Move code over to Rust 2018

### PR DESCRIPTION
**Pull request by blakehawkins**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.0.20
+0.0.21
```
Comment (rlupton20) [inline VERSION]: I'll cut a first release soon (probably version 0.1.0, or maybe just 1.0.0) once I have Travis building and deploying prebuilt binaries - as such no need to bump this.

File: src/main.rs [hunk 2]
```diff
@@ -1,10 +1,5 @@
 /// floki - the development container launcher
-#[macro_use]
-extern crate quicli;
-#[macro_use]
-extern crate failure;
-extern crate serde_yaml;
-extern crate uuid;
+#[macro_use] extern crate failure_derive;
```
Comment (rlupton20) [inline src/main.rs]: I'm not really clear on the change that happened in Rust 2018 that makes this possible - can you explain for my education?

Comment (blakehawkins) [reply src/main.rs]: Sure - take a look at this: https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html#no-more-extern-crate

And here's a summary, from the original blog post: https://blog.rust-lang.org/2018/12/06/Rust-1.31-and-rust-2018.html#module-system-changes

File: VERSION (hunk 1 above)
Comment (blakehawkins) [reply VERSION]: Fixed - feel free to squash on merge.

File: src/main.rs (hunk 2 above)
Comment (bossmc) [reply src/main.rs]: You don't even need to `extern crate` for macros any more, you can write

```
#[derive(failure::Fail)]
struct FooError(String);
```

Comment (rlupton20) [reply src/main.rs]: Thanks - that makes complete sense. Always wondered why extern crate was really necessary...

Will use this knowledge from now on!

Comment (rlupton20) [reply src/main.rs]: Thanks - that makes complete sense. Always wondered why extern crate was really necessary...

Will use this knowledge from now on!

**Comment by rlupton20** [comment]
Will merge this tomorrow when I can use my laptop to squash

**Comment by rlupton20** [comment]
Just needed to find squash on merge on my phone...

**Pull request merged**